safehttp/mux.go

```diff
@@ -164,6 +164,12 @@ func (m methodHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	defer func() {
+		if r := recover(); r != nil {
+			rw.ServerError(StatusInternalServerError)
+		}
+	}()
```

Comment:
Wouldn't this be handled by the `net/http` mux?

Reply:
But we might want to run some `After`-stages when `Before` panics, right? So then we need the control here.

Reply:
Add a comment about this.